cmd/wego/main.go

```diff
@@ -13,13 +13,54 @@ import (
 	fluxBin "github.com/weaveworks/weave-gitops/pkg/flux"
 )
 
+var VERSION = "0.0.0-dev.0"
+
 var options struct {
 	verbose bool
 }
 
 var rootCmd = &cobra.Command{
-	Use:   "wego",
-	Short: "Weave GitOps",
+	Use:           "wego [subcommand]",
+	Version:       VERSION,
```

Comment:
If you add Version here, it adds an implicit -v,--version flag and prints the version defined here.  This doesn't match what our `version` subcommand does so I think it will be confusing to the users.